src/main/views/template.njk

```diff
@@ -11,12 +11,12 @@
 
 {{ govukHeader({
         homepageUrl: "#",
-        serviceName: "Find a court or tribunal listing",
+        productName: "Find a court or tribunal listing",
```

Comment:
should this not be service name still? its moved it right across to the left now

Reply:
N. It has been decided that it is product name and the prototype has been changed accordingly. Please see prototype.

Reply:
just seen the prototype but it was changed though your suggestion?

Reply:
sorry what do you mean?

Reply:
well if youre steering the design then saying "look at the design" doesnt mean that its the right way

Reply:
@kurt-uk I don't know what you want to say with this. I haven't driven the design. This was decided by @johnoneill46 with Junior and they just send me a live example in the channel https://hmcts-reform.slack.com/archives/C027779QH40 where you are a member too. If you are not happy with the design speak to John and Junior then. Here I am just developing what they decide. I can raise my concerns but that's it. I am not the decision maker mate.

Reply:
Here im only talking about the `productName` vs the `serviceName` to which id been told that it was changed to match the code due to navigation concerns, i have no issues with the design and i think you're getting off topic a little. i asked you a question at the beginning because i was curious as to why we had changed from the recommended attribute
![Screenshot 2021-07-05 at 17 06 15](https://user-images.githubusercontent.com/26488566/124497801-56cb9880-ddb3-11eb-9acd-82303aee79f5.png)

Reply:
so are you happy to resolve this or do you need an action from me?

Reply:
could you revert to `serviceName` please? once its back the the original design itll be all ready to be tested 👍🏼

Reply:
code reverted